Gitpod not working

### Component

other

### Describe the bug

I tried to use the Gitpop linked in the Readme, but it doesn't work. Seems like the Go Version is to old.

### System Info

```shell
None
```


### Additional context

The log:
```
 HISTFILE=/workspace/.gitpod/cmd-4 history -r; {
cd docs/

} && {
pnpm install
pnpm build:woodpecker-plugins

} && {
pnpm start --port 4000

}
gitpod /workspace/woodpecker (master) $  HISTFILE=/workspace/.gitpod/cmd-4 history -r; {
> cd docs/
> 
> } && {
> pnpm install
> pnpm build:woodpecker-plugins
> 
> } && {
> pnpm start --port 4000
> 
> }
Scope: all 2 workspace projects
Lockfile is up to date, resolution step is skipped
Packages: +1137
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

   ╭─────────────────────────────────────────────────────────────────╮
   │                                                                 │
   │                Update available! 8.3.1 → 8.6.5.                 │
   │   Changelog: https://github.com/pnpm/pnpm/releases/tag/v8.6.5   │
   │                Run "pnpm add -g pnpm" to update.                │
   │                                                                 │
   │     Follow @pnpmjs for updates: https://twitter.com/pnpmjs      │
   │                                                                 │
   ╰─────────────────────────────────────────────────────────────────╯

Packages are cloned from the content-addressable store to the virtual store.
  Content-addressable store is at: /workspace/.pnpm-store/v3
  Virtual store is at:             node_modules/.pnpm
Downloading registry.npmjs.org/typescript/4.9.5: 11.6 MB/11.6 MB, done
Progress: resolved 1137, reused 0, downloaded 1135, added 1137, done
node_modules/.pnpm/core-js-pure@3.25.5/node_modules/core-js-pure: Running postinstall script, done in 69ms
node_modules/.pnpm/core-js@3.25.5/node_modules/core-js: Running postinstall script, done in 102ms
node_modules/.pnpm/esbuild@0.16.17/node_modules/esbuild: Running postinstall script, done in 137ms

dependencies:
+ @docusaurus/core 2.4.0
+ @docusaurus/preset-classic 2.4.0
+ @easyops-cn/docusaurus-search-local 0.35.0
+ @mdx-js/react 1.6.22
+ @svgr/webpack 6.5.1
+ clsx 1.2.1
+ esbuild-loader 2.21.0
+ file-loader 6.2.0
+ prism-react-renderer 1.3.5
+ react 17.0.2
+ react-dom 17.0.2
+ redocusaurus 1.6.2
+ url-loader 4.1.1

devDependencies:
+ @docusaurus/module-type-aliases 2.4.0
+ @tsconfig/docusaurus 1.0.7
+ @types/react 18.0.32
+ @types/react-helmet 6.1.6
+ @types/react-router-dom 5.3.3
+ typescript 4.9.5

The integrity of 3271 files was checked. This might have caused installation to take longer.
Done in 59.5s

> woodpecker@0.0.0 build:woodpecker-plugins /workspace/woodpecker/docs
> cd plugins/woodpecker-plugins && pnpm i && pnpm build

Scope: all 2 workspace projects
Done in 1.7s

> @woodpecker-ci/plugin-index@0.1.0 build /workspace/woodpecker/docs/plugins/woodpecker-plugins
> pnpm run style && tsc && tsc -p tsconfig.jsx.json


> @woodpecker-ci/plugin-index@0.1.0 style /workspace/woodpecker/docs/plugins/woodpecker-plugins
> mkdir -p dist/theme/ && cp src/theme/style.css dist/theme/style.css


> woodpecker@0.0.0 start /workspace/woodpecker/docs
> cd ../ && make docs && cd docs && docusaurus start "--port" "4000"

go generate cmd/cli/app.go
# github.com/woodpecker-ci/woodpecker/server/model
../../server/model/secret.go:128:18: undefined: errors.Join
note: module requires Go 1.20
# github.com/woodpecker-ci/woodpecker/shared/utils
../../shared/utils/context.go:28:25: undefined: context.WithCancelCause
note: module requires Go 1.20
# github.com/woodpecker-ci/woodpecker/pipeline/frontend/yaml/constraint
../../pipeline/frontend/yaml/constraint/constraint.go:245:68: undefined: errors.Join
note: module requires Go 1.20
cmd/cli/app.go:40: running "go": exit status 2
make: *** [Makefile:266: docs] Error 1
 ELIFECYCLE  Command failed with exit code 2.
gitpod /workspace/woodpecker/docs (master) $ 
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't [already an issue](https://github.com/woodpecker-ci/woodpecker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Checked that the bug isn't fixed in the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]
- [X] Check that this is a concrete bug. For Q&A join our [Discord Chat Server](https://discord.gg/fcMQqSMXJy) or the [Matrix room](https://matrix.to/#/#woodpecker:matrix.org).

--- a/.gitpod.yml
+++ b/.gitpod.yml
@@ -4,11 +4,13 @@ tasks:
       WOODPECKER_OPEN: true
       WOODPECKER_ADMIN: woodpecker
       WOODPECKER_HOST: http://host.docker.internal:8000
-      WOODPECKER_SECRET: "1234"
+      WOODPECKER_SECRET: '1234'
       WOODPECKER_GITEA: true
       WOODPECKER_DEV_WWW_PROXY: http://localhost:8010
       WOODPECKER_BACKEND_DOCKER_NETWORK: ci_default
     init: |
+      GO_VERSION=1.20
+      curl -fsSL https://dl.google.com/go/go$GO_VERSION.linux-amd64.tar.gz | tar xzs -C ~/
       go mod tidy
       mkdir -p web/dist
       touch web/dist/index.html
@@ -26,7 +28,7 @@ tasks:
   - name: Agent
     env:
       WOODPECKER_SERVER: localhost:9000
-      WOODPECKER_SECRET: "1234"
+      WOODPECKER_SECRET: '1234'
       WOODPECKER_MAX_WORKFLOWS: 1
       WOODPECKER_HEALTHCHECK: false
     command: |
@@ -102,12 +104,12 @@ ports:
 
 vscode:
   extensions:
-    - "golang.go"
-    - "EditorConfig.EditorConfig"
-    - "dbaeumer.vscode-eslint"
-    - "esbenp.prettier-vscode"
-    - "voorjaar.windicss-intellisense"
-    - "Vue.volar"
-    - "redhat.vscode-yaml"
-    - "davidanson.vscode-markdownlint"
-    - "streetsidesoftware.code-spell-checker"
+    - 'golang.go'
+    - 'EditorConfig.EditorConfig'
+    - 'dbaeumer.vscode-eslint'
+    - 'esbenp.prettier-vscode'
+    - 'voorjaar.windicss-intellisense'
+    - 'Vue.volar'
+    - 'redhat.vscode-yaml'
+    - 'davidanson.vscode-markdownlint'
+    - 'streetsidesoftware.code-spell-checker'

